src/styles/index.scss

```diff
@@ -1,6 +1,7 @@
 @import './constants/shared';
 
 body {
+  background-image: url('../../assets/pexels-photo-377058.jpeg');
   font-family: 'Roboto', sans-serif;
```

Comment:
![image](https://user-images.githubusercontent.com/12107969/32146934-e5e6ae20-bc9b-11e7-91e1-bc9b3d0ceba7.png)
Background image is cut off on slightly smaller screen size.